Documentation/botr/readytorun-format.md

```diff
@@ -326,6 +382,47 @@ This section marks region that contains thunks for ``READYTORUN_HELPER_DelayLoad
 helper. It is used by debugger for step-in into lazily resolved calls. It should not be required when 
 debuggers are able to handle debug info stored separately.
 
+## READYTORUN_SECTION_AVAILABLE_TYPES
+
+This section contains a native hashtable of all defined & export types within the compilation module. The key is a checksum of the full type name, the value is the exported type or defined type token row ID left-shifted by one and or-ed with bit 0 defining the token type:
```

Comment:
The key is the full type name.

Reply:
Hmm, I don't understand, can you please clarify? According to what I see in

https://github.com/dotnet/coreclr/blob/93675fbf467f54ab0f1f5d183c70750c9822c9ca/src/zap/zapreadytorun.cpp#L484

the key is the hash (should I have said 'hash' instead of 'checksum'?) of the full type name, not the type name itself.

Reply:
Renamed "checksum" to "hash" / "hashcode". Didn't change "hash of full type name" to "full type name" as I believe the vertex hashtable has the hash, not the full type name, as the key, but I'm prepared to be proven wrong and fix my mistake.

Reply:
https://en.wikipedia.org/wiki/Hash_table

The vertex table does not contain copy of the key inline; but it does not change the fact the key is the full type name.

Reply:
OK, I buy that, will fix in 3rd commit.